trollflow2/__init__.py

```diff
@@ -314,6 +314,53 @@ def sza_check(job):
             dpath.util.delete(product_list, '/product_list/%s' % area)
 
 
+def check_sunlight_coverage(job):
+    """Remove products with too low daytime coverage.
+
+    This plugins looks for a parameter called `min_sunlight_coverage` in the
+    product list, expressed in % (so between 0 and 100). Everything under it not
+    matching this sunlight coverage will be discarded.
```

Comment:
The last sentence needs some rewording to be more understandable. Maybe something like `If the sunlit fraction is less than configured, the affected products will be discarder.`?

Reply:
ok